modules/oe_content_policy/oe_content_policy.post_update.php

```diff
@@ -53,3 +53,24 @@ function oe_content_policy_post_update_00002_field_labels(array &$sandbox): void
     $field_config->save();
   }
 }
+
+/**
+ * Make the timeline field translatable and RDF fields not translatable.
+ */
+function oe_content_policy_post_update_00003_update_translatable_fields(array &$sandbox): void {
```

Comment:
We cannot change existing sites' translatability. So we need to move this update path to EWCMS.

Reply:
done here https://github.com/ec-europa/ewcms/pull/197